app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -96,7 +96,7 @@ description: |
   echo "Starting the subscription-manager registration process"
 
   # Set up subscription-manager
-  <%= snippet("subscription_manager_setup", variables: { subman_setup_scenario: 'provisioning' }).strip -%>
+  <%= indent(2) { snippet("subscription_manager_setup", variables: { subman_setup_scenario: 'provisioning' }).strip } -%>
```

Comment:
I think, skip1 is necessary so that the first line (# select...) is indented correctly.